functions/schedule-notifications.js

```diff
@@ -51,19 +49,24 @@ const sendPushNotificationToUsers = (userIds, payload) => {
 };
 
 exports.scheduleNotifications = functions.pubsub.topic('schedule-tick').onPublish(() => {
-  const todayDay = moment().utcOffset(TIMEZONE).format('YYYY-MM-DD');
+  const notificationsConfigPromise = admin.database().ref(`/notifications/config`).once('value');
+  const schedulePromise = admin.database().ref(`/schedule`).once('value');
 
-  return admin.database().ref(`/schedule`).once('value')
-    .then(scheduleSnapshot => {
+  return Promise.all([notificationsConfigPromise, schedulePromise])
+    .then(([notificationsConfigSnapshot, scheduleSnapshot]) => {
+      const notificationsConfig = notificationsConfigSnapshot.val();
       const schedule = scheduleSnapshot.val();
+      console.log('notificationsConfig', notificationsConfig);
+      console.log('schedule', schedule);
```

Comment:
do we need to console log this info, or you forgot to remove after debugging?

Reply:
no,  forgot to remove ;)